less/mixins.less

```diff
@@ -1 +1,14 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+
+.displayFlex(@justify,  @align-i) {
+    display: flex;
+    justify-content: @justify;
+    align-content: @align-i;
+}
+
+.displayBtn(@align-i, @justify, @background) {
+    display: flex;
+    align-items: @align-i;
+    justify-content: @justify;
+    background-color: @background;
+}
```

Comment:
one thing that could be cool in here is to add in that hover, so you can add a background and border color and the colors on hover then all you buttons have that awesome feedback built right in